kivy/core/window/__init__.py

```diff
@@ -933,6 +933,34 @@ def _set_top(self, value):
     the position set in :class:`~kivy.config.Config`.
     '''
 
+    def _get_opacity(self):
+        return self._get_window_opacity()
+
+    def _set_opacity(self, opacity):
+        self._set_window_opacity(opacity)
```

Comment:
Change to `return self._set_window_opacity(opacity)` so that window specific implementation can decide what to do.

Reply:
Done.